packages/eyes/src/eyes-test/eyes.ts

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const fs = require('fs');
+const glob = require('glob');
+const chalk = require('chalk');
+const dotenv = require('dotenv');
+const {JSDOM} = require('jsdom');
+const {makeVisualGridClient, initConfig} = require('@applitools/visual-grid-client');
+const domNodesToCdt = require('@applitools/visual-grid-client/src/browser-util/domNodesToCdt');
+const {TestFailedError, TestResults} = require('@applitools/eyes.sdk.core');
+
+dotenv.config();
+
+function getGridClientConfig() {
+  const projectName = 'kieran-grid-test';
```

Comment:
Let's take this from the `package.json` as we did before. And create a task on leankit for github integration, which will allow us to automatically set the `branchName`.